### PR TITLE
Replace deprecated include with include_tasks

### PR DESCRIPTION
**Pull request by Zokormazo**

### CODE REVIEW

**Comment by scuben** [comment]
Hi @Zokormazo 

Shouldn't it be changed to `import_task` instead of `include_task`? `include_tasks` is only needed if you include another task file depending on runtime (e.g. a variable).

File: config/after_symlink_shared.yml [hunk 1]
```diff
@@ -1,17 +1,17 @@
-- include: "../../cbrunnkvist.ansistrano-symfony-deploy/config/steps/composer.yml"
+- include_tasks: "../../cbrunnkvist.ansistrano-symfony-deploy/config/steps/composer.yml"
```
Comment (scuben) [inline config/after_symlink_shared.yml]: change al `import_task` to `import_task`.

**Comment by scuben** [comment]
Just saw your other PR https://github.com/cbrunnkvist/ansistrano-symfony-deploy/pull/19. Guess this will be obsolete then?

**Comment by Zokormazo** [comment]
Yep, you're right, it should be an import since it's a static include. I'll
update the PR ASAP

**Comment by Zokormazo** [comment]
They should two independent PR. This one updates the existing deprecated include module references. 

The #19 PR adds ansistrano style Hooks to ansistrano-symfony-deploy
Once this one is merged, I'll update the other PR to adapt it to HEAD & use import_tasks instead of include_tasks

**Comment by scuben** [comment]
@Zokormazo Thank you for your contribution!

**Pull request merged**